src/countOccurrences.js

```diff
@@ -18,7 +18,16 @@
  *
  */
 function countOccurrences(phrase, letter) {
-  // write code here
+  let count = 0;
+  const phraseWork = phrase.toLowerCase();
```

Comment:
what does work mean? you make it lowercase
```suggestion
  const phraseLowerCase = phrase.toLowerCase();
```